tsdb/engine/tsm1/engine.go

```diff
@@ -938,16 +940,15 @@ func (e *Engine) WritePoints(points []models.Point) error {
 func (e *Engine) DeleteSeriesRange(itr tsdb.SeriesIterator, min, max int64) error {
 	var disableOnce bool
 
-	// Ensure that the index does not compact away the measurement or series we're
-	// going to delete before we're done with them.
-	if tsiIndex, ok := e.index.(*tsi1.Index); ok {
```

Comment:
This block was needed to prevent TSI compactions compacting away a series while we're iterating over it.  Is this removed because iteration is over the series file now?